dir.props

```diff
@@ -197,4 +198,7 @@
     <PackageIndex Include="$(PackageIndexFile)" />
   </ItemGroup>
 
+  <!-- Use Roslyn Compilers to build -->
+  <Import Project="$(RoslynPropsFile)" Condition="'$(RunningOnUnix)'!='true' and Exists('$(RoslynPropsFile)') and '$(UseRoslynCompilers)'!='false'" />
```

Comment:
I did a local build in a VS2017 dev command prompt.  Is there additional validation you want?